app/facades/food_facade.rb

```diff
@@ -0,0 +1,13 @@
+# app/facades/food_facade.rb
+class FoodFacade
+  def search_foods(food_name)
+    api_key = Rails.application.credentials.dig(:food_data, :key) 
```

Comment:
Ok cool, we have a facade! Most of what you're doing here though belongs in a service. As a review: 

* a Facade is like a special "helper" controller, we can do things like `JSON.parse` and turn data into PORO objects here. 
* A Service is what we use to connect to an external API. It deals with things like Rails Credentials, `Faraday.new(..)` and sending data to the API.